testing/sdk_diag.py

```diff
@@ -162,6 +167,15 @@ def _dump_plans(item: pytest.Item, service_name: str):
             f.write('\n') # ... and a trailing newline
 
 
+def _dump_threads(item: pytest.Item, service_name: str):
+    threads = sdk_cmd.service_request('GET', 'v1/debug/threads')
```

Comment:
There is a missing parameter here.

The `pylint -E testing` output:
```
************* Module sdk_diag
E:172,14: No value for argument 'service_path' in function call (no-value-for-parameter)
```
This should be:
```python
threads = sdk_cmd.service_request('GET', service_name, 'v1/debug/threads')
```

I have corrected this in #2532.